src/NUnitFramework/nunitlite/OutputWriters/NUnit2XmlOutputWriter.cs

```diff
@@ -89,7 +89,7 @@ private void InitializeXmlFile(ITestResult result)
             xmlWriter.WriteAttributeString("total", summary.TestCount.ToString());
             xmlWriter.WriteAttributeString("errors", summary.ErrorCount.ToString());
             xmlWriter.WriteAttributeString("failures", summary.FailureCount.ToString());
-            var notRunTotal = summary.SkipCount + summary.FailureCount + summary.InvalidCount;
+            var notRunTotal = summary.IgnoreCount + summary.ExplicitCount + summary.SkipCount + summary.InvalidCount;
```

Comment:
The old code didn't make sense to me. Why are failed tests not run? This seems more correct to me.

Reply:
The ResultSummary class has a method called ```NotRunCount``` which looks like:  
```
 public int NotRunCount
        {
            get { return InvalidCount + SkipCount + IgnoreCount + ExplicitCount;  }
        }
```
Seems like the same as is expressed here. Perhaps better to use that?

Reply:
Good idea @OsirisTerje, I made the change.